package.json

```diff
@@ -137,7 +137,7 @@
         "ng-morph": "4.5.4",
         "ng-packagr": "16.2.3",
         "ngx-highlightjs": "10.0.0",
-        "nx": "18.3.5",
+        "nx": "19.5.3",
```

Comment:
Update also this dependency 

https://github.com/taiga-family/taiga-ui/blob/d631e0a1b790c87ee5586b11e1ad61534c4a8cf3/projects/demo-cypress/package.json#L5

Reply:
fixed